### ISSUE TITLE
New concourse version seems to break the docker-compose file

### ISSUE THREAD

**Issue by BenGale** [starkandwayne/concourse-tutorial#153]
Sorry for not tonnes off detail here, but I found an issue where if you run docker-compose up on a clean docker instance (no cached images), it now grabs the v4 of concourse. There seems to be some issue with the user roles:

concourse_1     | default team auth not configured: No auth methods have been configured.

If I update the docker-compose file to point at the latest v3 tag:

image: concourse/concourse:3.14.1

then it all comes up fine as before👍

**Comment by drnic** [comment]
Thanks. I'll add a bespoke `docker-compose.yml` into this project fixed to v3.1.1 until I've got time to investigate the new v4 auth/teams system and update the tutorial,.

**Comment by drnic** [comment]
```plain
    wget https://raw.githubusercontent.com/starkandwayne/concourse-tutorial/master/docker-compose.yml
    docker-compose up -d
    ```
